src/pages/wallet-details/wallet-details.scss

```diff
@@ -69,49 +69,44 @@ page-wallet-details {
   .wallet-info {
     font-weight: 500;
     color: color($colors, light);
-    height: 30px;
+    height: 2.5rem;
+    padding-left: 0.5rem;
+    span {
+      margin: 0 3px;
+      padding-top: 2px;
+      float: left;
+    }
     img {
       float: left;
-      margin-right: 4px;
     }
     .testnet {
       height: 2.5rem;
       width: 2.5rem;
-      margin-top: 3px;
     }
     .testnet-text {
-      color: color($colors, light);
-      height: 3rem;
-      width: 5rem;
+      height: 2.5rem;
+      width: 4.5rem;
+      margin-right: 5px;
     }
     .read-only {
-      height: 3rem;
-      width: 2rem;
+      height: 2.5rem;
+      width: 1.5rem;
+      margin-right: 5px;
     }
     .read-only-text {
-      height: 3rem;
-      width: 6rem;
+      height: 2.5rem;
+      width: 5rem;
+      margin-right: 5px;
     }
     .auditable-text {
-      height: 3rem;
+      height: 2.5rem;
       width: 5rem;
     }
-    .custom-bws {
-      height: 3.2rem;
-      width: 3.2rem;
-    }
-    .wallet-type {
-      display: block;
-      float: left;
-      margin: 0 3px;
-      padding-top: 5px;
-    }
     ion-spinner {
       float: right;
       width: 20px;
       height: 20px;
-      margin: 5px 5px 0 0;
-      * {
+      &.spinner-crescent circle {
```

Comment:
`spinner-crescent` is for Android only. Using `*` will work on iOS also.

Reply:
Done. Thanks!